packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -308,6 +308,8 @@ export default class TradeStore extends BaseStore {
 
     @action.bound
     async prepareTradeStore() {
+        // show errors if any
+        this.root_store.common.setShouldSkipRaiseError(false);
```

Comment:
Assuming that we get `InvalidToken` response while we are in trade page. Will this still be called?